test/service/test_app_logging.py

```diff
@@ -21,62 +25,229 @@
+                        'type': 'terms'
+                    },
+                    'dataUseRestriction': {
+                        'terms': [
+                            {
+                                'term': None,
+                                'count': 1
+                            }
+                        ],
+                        'total': 1,
+                        'type': 'terms'
+                    },
+                    'project': {
+                        'terms': [
+                            {
+                                'term': 'Single of human pancreas',
+                                'count': 1,
+                                'projectId': ['e8642221-4c2c-4fd7-b926-a68bce363c88']
+                            }
+                        ],
+                        'total': 1,
+                        'type': 'terms'
+                    },
+                    'sampleDisease': {
+                        'terms': [
+                            {
+                                'term': 'normal',
+                                'count': 1
+                            }
+                        ],
+                        'total': 1,
+                        'type': 'terms'
+                    },
+                    'nucleicAcidSource': {
+                        'terms': [
+                            {
+                                'term': 'single cell',
+                                'count': 1
+                            }
+                        ],
+                        'total': 1,
+                        'type': 'terms'
+                    },
+                    'assayType': {
+                        'terms': [
+                            {
+                                'term': None,
+                                'count': 1
+                            }
+                        ],
+                        'total': 0,
+                        'type': 'terms'
+                    },
+                    'instrumentManufacturerModel': {
+                        'terms': [
+                            {
+                                'term': 'Illumina NextSeq 500',
+                                'count': 1
+                            }
+                        ],
+                        'total': 1,
+                        'type': 'terms'
+                    },
+                    'institution': {
+                        'terms': [
+                            {
+                                'term': 'Farmers Trucks',
+                                'count': 1
+                            },
+                            {
+                                'term': 'University',
+                                'count': 1
+                            }
+                        ],
+                        'total': 1,
+                        'type': 'terms'
+                    }
+                }
+            }
+        )[:1024]  # … upto the first 1024 characters.
```

Comment:
```suggestion
        )
```

Consider returning the full value here, then slicing at the call sites where a comment won't be necessary since it'll be clear from the nearby log lines why the value is sliced.